Brocfile.js

```diff
@@ -33,20 +33,26 @@ var es5 = new Babel(lib, {
   blacklist: ['es6.modules']
 });
 
+function rollupConfig(entry) {
+  return new Rollup(es5, {
+    rollup: {
+      entry: 'lib/' + entry,
+      cache: 'BUILD:es6-promise',
```

Comment:
https://github.com/chadhietala/broccoli-rollup/blob/master/src/index.js
Ok just saw this plugin automatically does it for me. I was surprised why was I getting good speeds, the plugin was overriding the cache variable for me. 😜